FMDB.podspec

```diff
@@ -26,24 +31,28 @@ Pod::Spec.new do |s|
   s.subspec 'standalone' do |ss|
     ss.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DFMDB_SQLITE_STANDALONE' }
     ss.dependency 'sqlite3'
-    ss.source_files = 'src/fmdb/FM*.{h,m}'
-    ss.exclude_files = 'src/fmdb.m'
-  end
-
-  # build with FTS support and custom FTS tokenizer source files
+    ss.dependency 'FMDB/common'
+	end
+	
+  # build the latest stable version of sqlite3 
+	# with FTS support and custom FTS tokenizer source files
   s.subspec 'standalone-fts' do |ss|
-    ss.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DFMDB_SQLITE_STANDALONE' }
-    ss.source_files = 'src/fmdb/FM*.{h,m}', 'src/extra/fts3/*.{h,m}'
-    ss.exclude_files = 'src/fmdb.m'
+    ss.dependency 'FMDB/standalone'
     ss.dependency 'sqlite3/fts'
+    ss.source_files = 'src/extra/fts3/*.{h,m}'
   end
 
   # use SQLCipher and enable -DSQLITE_HAS_CODEC flag
   s.subspec 'SQLCipher' do |ss|
     ss.dependency 'SQLCipher'
-    ss.source_files = 'src/fmdb/FM*.{h,m}'
-    ss.exclude_files = 'src/fmdb.m'
-    ss.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DSQLITE_HAS_CODEC -DHAVE_USLEEP=1' }
-  end
-  
+  	ss.dependency 'FMDB/common'
```

Comment:
Though it wouldn't affect how this functions, the indentation on this is wrong. it should like up with:
```
ss.dependency 'SQLCipher'
ss.dependency 'FMDB/common'
ss.xcconfig = ...
```